### ISSUE TITLE
RPR 2.5.20 Texture issues

### ISSUE THREAD

**Issue by Aliprandini** [GPUOpen-LibrariesAndSDKs/RadeonProRenderBlenderAddon#117]
Some textures issues while render with RPR Full.
See attachment. 

Also, RPR Full mode much slower than legacy mode (2x-4x slowly).

Mac OS 10.5.7, 5700 XT, Blender 2.91 beta, RPR 2.5.20

![full](https://user-images.githubusercontent.com/73755453/97786281-f7820c80-1bc3-11eb-940d-05b6f56dcc4b.png)
![Legacy](https://user-images.githubusercontent.com/73755453/97786283-f8b33980-1bc3-11eb-8ca7-6c67d1dafb6b.png)

[RyzenGraphic_27_prorender.blend.zip](https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderBlenderAddon/files/5469543/RyzenGraphic_27_prorender.blend.zip)

**Comment by bsavery** [comment]
hey thanks for the scene.  We have some known optimizations pending around transparent surfaces and refraction, which clearly your scene uses extensively ;)

So it's a good test case, but we're already working on it.  Thanks!

**Comment by bsavery** [comment]
So I'm going to close the issue, here's a few things to note which I claim weight things in rpr 1 for timing here

1.  We had a slowdown with RPR 2 with transparent objects which is now fixed.  
2.  You have adaptive sampling turned on in RPR1 which RPR2 does not support yet.
3.  With low sampling counts things might be exaggerated.  
4.  RPR2 works most efficient with sample counts of multiples of 32.  (you have 100 which is 3*32 + 4)
5.  We enabled better transparent (colored) shadows in RPR2 .  See the ground under the color circle
6.  For a given number of samples the noise here is much much better in rpr 2.  You have denoise turned on which hides this a bit but here's an example:

RPR 1
![rpr1](https://user-images.githubusercontent.com/1892220/99128420-97c23300-25bf-11eb-9fae-d331587e8aba.png)
RPR 2
![rpr2](https://user-images.githubusercontent.com/1892220/99128429-9c86e700-25bf-11eb-8e92-d4f08f68a989.png)
